modules/apps/transfer/types/keys.go

```diff
@@ -29,9 +29,12 @@ const (
 	// AllowAllPacketDataKeys holds the string key that allows all memo strings in authz transfer messages
 	AllowAllPacketDataKeys = "*"
 
-	KeyTotalEscrowPrefix = "totalEscrowForDenom"
+	KeyTotalEscrowPrefix  = "totalEscrowForDenom"
+	KeyTotalForwardPrefix = "totalForwardForDenom"
 
 	ParamsKey = "params"
+
+	KeyPacketForwardPrefix = "forwardedPacket"
```

Comment:
a bit of inconsistency we have here. Some keys are defined below as consts with a byte prefix, while other keys (total escrow and this packet forward prefix) use string prefixes. The byte prefixes use a prefix store in the keeper setting code while the string prefixes use helper functions. In general would prefer store prefixes to be grouped together in one section so it is easy to understand what keys exist for state storage

Reply:
Let's group them together in a follow up PR